code/modules/camera/camera.dm

```diff
@@ -0,0 +1,341 @@
+/obj/machinery/camera
+	name = "security camera"
+	desc = "A small, high quality camera equipped with face and ID recognition. It is tied into a computer system, allowing AI and those with access to watch what occurs through it."
+	icon = 'icons/obj/monitors.dmi'
+	icon_state = "camera"
+	deconstruct_flags = DECON_SCREWDRIVER | DECON_WELDER | DECON_WIRECUTTERS | DECON_MULTITOOL
+	text = ""
+
+	/// Used by camera monitors to display certain cameras only
+	var/network = "SS13"
+	/// Used by autoname: EX "security camera"
+	var/prefix = "security"
+	/// Used by autoname: EX "camera - west primary hallway"
+	var/uses_area_name = FALSE
+	// These stack: EX "security camera - west primary hallway"
```

Comment:
comment doesn't make sense, is this for the prior thing?

Reply:
yeah, updated the comment to clarify